workshop.wwwapi/Repository/IRepository.cs

```diff
@@ -5,6 +5,8 @@ namespace workshop.wwwapi.Repository
     public interface IRepository
     {
         Task<IEnumerable<Patient>> GetPatients();
+
+        Task<Patient> GetPatientByID(int id);
```

Comment:
return `Patient?` which is nullable in case you don't find patient...

Reply:
Resolved this as well